src/main/java/com/api/util/ApiSecurity/ApiList.java

```diff
@@ -36,17 +35,22 @@ public String toString(String delimiter, Boolean sort, Boolean quote, Boolean is
 		
 		/* Sort key first then value*/
 		if (sort){
-			list = this.stream()
-					.sorted((Entry<String,String> l1, Entry<String,String> l2) -> 
-					{
-						return l1.getKey().equals(l2.getKey()) ? l1.getValue().compareTo(l2.getValue())
-									: l1.getKey().compareTo(l2.getKey());
-					})
-					.map(e -> (null== e.getValue() || (null!= e.getValue() && e.getValue().isEmpty()) && isBaseString) ? e.getKey() : String.format(format, e.getKey(), e.getValue())  )
-					.collect(Collectors.toList());
+			List<Entry<String, String>> toSort = new ArrayList<>(this);
+			toSort.sort((Entry<String, String> l1, Entry<String, String> l2) ->
+					l1.getKey().equals(l2.getKey()) ? l1.getValue().compareTo(l2.getValue()) : l1.getKey().compareTo(l2.getKey()));
+			List<String> result = new ArrayList<>();
+			for (Entry<String, String> e : toSort) {
+				String s = null == e.getValue() || e.getValue().isEmpty() && isBaseString ? e.getKey() : String.format(format, e.getKey(), e.getValue());
```

Comment:
string assignment to null is not necessary

Reply:
Also please update the version in build.gradle , pom.xml and README.md for the new snapshot version number
 <version>1.3.X-SNAPSHOT</version> 

Supposedly to be 1.3.5 already based on the pom and gradle files.
Please help to fix the readme version as it is still outdated at 1.3.2, also put support for Java 1.7 in the guide section